Template-2.md

```diff
@@ -0,0 +1,249 @@
+  > The **permalink** title should be in the format: **/mypage/**, instead of _mypage.md_  
+  7. Once you have made these changes, scroll down to the bottom of your screen. At the left side of the screen, click on the green button that says **Commit changes.**<BR>
+  > This action saves all of your changes to your Page.
+
+# TEMPLATES
+
+# Example Outline Template
+
+Now that you have created a new Page to start your guide, this **Example Outline** will help you to structure and develop content for your guide.
+
+  1. If you have not already done so, copy and paste this **Example Outline** into your new Page.
+
+## Overview
+
+This introductory text should give a brief overview of the subject matter covered in your guide. In addition to the subject-matter overview, it may include information on the intended audience, the intended outcome of the guide, and any other information that would help the user to understand the guide.
+
+## Assumptions 
+
+This section should be a bullet list that explains any assumptions that must be understood before the user can begin the procedures in the guide. This section may also include conditions that must be met before the user can begin the procedures. Alternate titles may be: **_Preconditions_** (i.e., list any conditions that must be met) or **_Hardware and Software_Requirements** (i.e, list any hardware or software that must be obtained and/or configured).
+
+## Before You Get Started
+
+This section will tell the user what he/she needs to do to prepare before starting a set of procedures. 
+
+## Title of Procedure 1
+
+This introductory text should give any user-friendly background information; explain the purpose of the procedure; and give any context that the user might need before diving into the steps. The text within each step should walk the user directly through exactly what they need to do to complete the procedure.
+
+  1. Step 1 of procedure 1.
+  2. Step 2 of procedure 1.
+  
+## Title of Procedure 2
+
+This introductory text should give any user-friendly background information; explain the purpose of the procedure; and give any context that the user might need before diving into the steps. The text within each step should walk the user directly through exactly what they need to do to complete the procedure.
+
+  1. Step 1 of the procedure 2.
+  2. Step 2 of the procedure 2.
+  
+  > Repeat procedural sections, as needed.
+
+# Markdown Template
+
+This **Markdown Template** will help you to use Markdown syntax to format the content for your guide. You can learn more about Markdown formatting by browsing to the "Mastering Markdown" guide from Github: https://guides.github.com/features/mastering-markdown/.
+
+  1. If you have not already done so, copy and paste this **Markdown Template** into your new Page.  
+
+## Headings
+
+The required Heading formats for the www.IDManagement.gov website are as follows:
```

Comment:
This is confusing.  This isn't the idmanagement.gov website, and we're using a different css and heading style in the playbooks.